gpjax/kernels/stationary/changepoint.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+"""
+AUTHOR: Jake Yeung (jakeyeung@gmail.com)
+CREATED ON:  2023-07-28
+LAST CHANGE: see git log
+LICENSE:     Apache License
+"""
+
+from dataclasses import dataclass
+
+from beartype.typing import Union
+import jax
+import jax.numpy as jnp
+from jaxtyping import Float
+import tensorflow_probability.substrates.jax.bijectors as tfb
+
+import gpjax as gpx
+from gpjax.base.param import param_field
+from gpjax.typing import (
+    Array,
+    ScalarFloat,
+)
+
+
+@dataclass
+class ChangePointRBF(gpx.kernels.AbstractKernel):
+    r"""The change point kernel for two RBFs."""
```

Comment:
Having some further details here would be helpful. At the minimum, explain what the kernel does and define the arguments required for initialisation.

Reply:
The initialization is now a list of two kernels, which hopefully simplifies the details here.